pom.xml

```diff
@@ -56,6 +56,11 @@
       <name>PentahoNexus</name>
       <url>https://nexus.pentaho.org/content/groups/omni</url>
     </repository>
+    <repository>
+      <id>apache</id>
+      <name>Apache</name>
+    <url>https://repository.apache.org/content/repositories/public/</url>
```

Comment:
Just wondering if we need this as Jena 3.17.0 is also in Maven Central

Reply:
Removed it. Think I added it when I was trying to use Jena 4.